_ml-commons-plugin/cluster-settings.md

```diff
@@ -103,14 +103,14 @@ plugins.ml_commons.monitoring_request_count: 100
 - Default value: `100`
 - Value range: [0, 10,000,000]
 
-## Upload model tasks per node
+## Register model tasks per node
 
-Controls how many upload model tasks can run in parallel on one node. If set to `0`, you cannot upload models to any node.
+Controls how many register model tasks can run in parallel on one node. If set to `0`, you cannot run register model tasks on any node.
```

Comment:
Just wondering. What do you think about hyphens for these compounds?
register-model tasks.
deply-model tasks.

Reply:
Let me think about that one.